__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const devData = require('../db/data/test-data/index');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+
+const app = require('../app');
+const request = require('supertest');
+
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+
```

Comment:
This doesn't need to be an if statement - it can just be db.end() inside the callback.